game/src/debug/routes.rs

```diff
@@ -404,10 +403,14 @@ impl State<App> for AllRoutesExplorer {
                                 .max()
                                 .unwrap() as f64;
                             for (r, before, after) in comparisons {
-                                if after < before {
-                                    colorer.add_r(r, less.eval((before - after) as f64 / diff));
-                                } else if before < after {
-                                    colorer.add_r(r, more.eval((after - before) as f64 / diff));
+                                match after.cmp(&before) {
```

Comment:
This one's interesting. Much more verbose, but the `match` makes the 3 cases very explicit, and it's harder to mess up the ordering in the predicate. Still have to get the math in the handlers right.

(I'm fine with this, just commenting)

Reply:
The math could be simplified with `abs`. While this technically add an unnecessary comparison, I expect it to be optimized away.